exercises/5_OOP-and-Intents/oop/src/ClarinetPlayer.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Created by c4q-rosmary on 4/30/15.
+ */
+
+//Create the concrete class ClarinetPlayer, which extends Musician.
+public class ClarinetPlayer<T> extends Musician
+{
+    protected Clarinet clarinet;
+
+
+    //The ClarinetPlayer constructor should take in a Clarinet, and save it in a field called myClarinet
+    public ClarinetPlayer (Clarinet clarinet) {
+        this.clarinet = clarinet;
+
+    }
+
+    //Add a method toggle_reed, which puts a reed on the Clarinet if it doesn't have one, and takes the reed off it if does.
+    public void toggle_reed(){
+        if (clarinet.has_reed()) {//if true
```

Comment:
The `// if true` comment is redundant for this code block. In general, comments that explain _why_ are more helpful than comments that explain _what_.